types/runtime/index.d.ts

```diff
@@ -54,6 +56,9 @@ export class View {
   addResizeListener(handler: ResizeHandler): View;
   removeResizeListener(handler: ResizeHandler): View;
   tooltip(handler: TooltipHandler): View;
+
+  getState(options?: any): any;
```

Comment:
Can you make the options more specific?

Reply:
You mean `object`? or even more specific?

Reply:
More specific

Reply:
Done.